### ISSUE TITLE
Pattern and Output Folders don't work when nested under a folder eg. Fabric\Patterns

### ISSUE THREAD

**Issue by wallykroeker** [chasebank87/mesh-ai#16]
When I first set it up I decided to get fancy and nest everything under my Fabric folder.  When I did that it came back with an error that I it could not find the pattern I chose.  When I created folders for output and patterns in the root then it worked as intended.

**Comment by chasebank87** [comment]
I will look into this. Thanks for the heads up

**Comment by wally-kroeker** [comment]
I was logged in with my work account when I posted this. posting with my personal account just to track this.

**Comment by grizzlycode** [comment]
I'm having a similar issue as OP. I have it nested and it says it can't find the pattern.

**Comment by chasebank87** [comment]
Im trying to duplicate this issue, and I havent been able to. Can you show me a screenshot of the setting and the folder location in your vault. This will help me replicate.

**Comment by grizzlycode** [comment]
Not OP, but here is my config. I get the "Error: Pattern ‘summarize not found in custom or fabric patterns folders" no matter which pattern I pick. 

![aimesh1](https://github.com/user-attachments/assets/8bfaa882-7b7d-4628-93ad-b83344df6f36)
![aimesh2](https://github.com/user-attachments/assets/b8de7926-700a-417f-8f14-35a239fb280e)

**Comment by chasebank87** [comment]
Try forward slashes pleaseSent from my iPhoneOn Oct 2, 2024, at 5:06 PM, grizzlycode ***@***.***> wrote:﻿
Not OP, but here is my config. I get the "Error: Pattern ‘summarize not found in custom or fabric patterns folders" no matter which pattern I pick.
aimesh1.png (view on web)
aimesh2.png (view on web)

**Comment by grizzlycode** [comment]
I'm currently on Windows so that is why I used backslashes for the path. Which works to download the patterns. 

I changed to forward slashes and its processing the request. I'm currently using Ollama to process the request I can see the activity and it does eventually  complete it. However, it does seem to take quite a bit of time to process the request. I'll double check that part on my end. 

So I guess the slash orientation mattters no matter if its Windows path as well. I'm not the OP, but I seems to be working now at least in my case.

**Comment by chasebank87** [comment]
Yeah thats pretty normal unless you have a beast of a computer. One thing I would check if I were you, is to try lmstudio because it may be that the model isnt using the GPU like it should. But if you are running larger models past 8b parameters its going to be slow no matter what most likely.
